### PR TITLE
OrganizationApiController: adding unit tests

### LINKED ISSUE #615: OrganizationApiController: adding unit tests
I'm currently working on this

### PR DESCRIPTION
**Pull request by mgmccarthy**
Fixes #615

### CODE REVIEW

**Comment by tonysurma** [comment]
did a quick review looks good to me, but if @mheggeseth @MisterJames can glance at it before I merge that would be great

File: AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Organizations/OrganizationContactQueryHandler.cs [hunk 1]
```diff
@@ -17,24 +17,21 @@ public OrganizationContactQueryHandler(AllReadyContext context)
...
-               .SingleOrDefault();
-            if (t == null)
-            {
+
+            var organization = _context.Organizations.AsNoTracking()
+                .Include(l => l.Location).ThenInclude(pc => pc.PostalCode)
+                .Include(oc => oc.OrganizationContacts).ThenInclude(c => c.Contact)
+               .SingleOrDefault(o => o.Id == message.OrganizationId);
+
+            if (organization == null)
```
Comment (BillWagner) [inline AllReadyApp/Web-App/AllReady/Areas/Admin/Features/Organizations/OrganizationContactQueryHandler.cs]: Please leave the { } around the if and else clauses.

**Comment by BillWagner** [comment]
Comments added. Once updated, please merge.

**Comment by mgmccarthy** [comment]
pinging @BillWagner. All curly braces added back in. We should be good to go on this.

**Pull request merged**